File: Question/26_Remove_Duplicates_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int ans = nums.size();
        if(ans <=1) return ans;
        int left = 0;
        int right = 1;
        while(right < nums.size()) {
            if(nums[left] == nums[right]) {
                ++right;
                --ans;
            }
            else
                nums[++left] = nums[right++];
        }
        return ans;
    }
};

File: Question_Java/26_Remove_Duplicates_from_Sorted_Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums.length <= 1) return nums.length;
        int cur = 0;
        for (int i = 1; i < nums.length; ++i) {
            if (nums[i] == nums[cur]) continue;
            nums[++cur] = nums[i];
        }
        return cur + 1;
    }
}